octopusdeploy_framework/resource_aws_elastic_container_registry.go

```diff
@@ -39,13 +39,14 @@ func (r *awsElasticContainerRegistryFeedTypeResource) Configure(_ context.Contex
 }
 
 func (r *awsElasticContainerRegistryFeedTypeResource) Create(ctx context.Context, req resource.CreateRequest, resp *resource.CreateResponse) {
+	tflog.Info(ctx, "creating Aws Elastic Container Registry")
```

Comment:
we have a util method for this...`util.Create(...)`

Reply:
fixed, plus a few other places